src/components/layouts/SideBar.js

```diff
@@ -68,10 +68,25 @@ const Sidebar = () => {
             <NavItem item="resources" icon="fas fa-chalkboard-teacher " />
             <NavItem item="roadmaps" icon="fas fa-map-signs" />
           </aside>
+
+          {/* <!-- discord buton --> */}
+          <div className="absolute bottom-10 w-full flex justify-center pr-4">
+            <button className="bg-blue-500 flex items-center justify-center p-3 xl:px-6 rounded transition-all duration-200 bg-gradient-to-br hover:from-purple-500 hover:to-indigo-500 hover:text-white text-base xs:text-left md:text-center lg:text-left">
+              <i class="fab fa-discord text-2xl mr-1 xs:mr-3 md:mr-0 xl:mr-3 xl:text-base text-center"></i>
+              <a
+                href="https://discord.com/invite/FYSQUEw6xP"
+                target="_blank"
+                rel="noopener noreferrer"
+                className="xs:inline-block md:hidden xl:inline-block"
+              >
+                Join Discord
+              </a>
+            </button>
```

Comment:
I would like to suggest one thing instead of putting an anchor tag in the button, convert an anchor to a button and put an icon inside the anchor tag.
Remove the button tag and copy the tailwind classes from the button and paste it to an anchor tag.
Does it make sense to you?

Reply:
What are the advantages of converting anchor to a button?
Just curious to know.

Reply:
Not a very big advantage, we don't need to use one extra tag.

Reply:
I have made changes and the button is now converted into a anchor tag

Reply:
Button looks good on desktop but not on mobile and tablet.
In tablet, it should be square and in mobile, it should take up the whole width of the mobile menu. Just fix these two issues.
<img width="543" alt="Screenshot 2021-09-24 at 7 54 39 PM" src="https://user-images.githubusercontent.com/57035408/134691000-d3176219-8255-4b3d-8829-2b198a8452ca.png">
<img width="83" alt="Screenshot 2021-09-24 at 7 56 13 PM" src="https://user-images.githubusercontent.com/57035408/134691092-3f25843b-bc5b-4443-b8e7-6407b3914f1f.png">

Reply:
Will fix it